javascript/operators/object_initializer.json

```
@@ -178,7 +178,7 @@
                 "version_added": false
               },
               "nodejs": {
-                "version_added": true
+                "version_added": "6.0.0"
```

Also possibly earlier? https://node.green/#ES2015-syntax-object-literal-extensions

```suggestion
                "version_added": "4.0.0"
```